distribution/pull_v2_windows.go

```diff
@@ -74,7 +76,9 @@ func filterManifests(manifests []manifestlist.ManifestDescriptor, os string) []m
 		// TODO: Consider filtering out greater versions, including only greater UBR
```

Comment:
Might want to remove this TODO, as we don't want to filter greater UBR here anymore as the returned error would be just that there is no matching image in the manifest, rather than an incompatible version.

Reply:
Done, update pushed.